### DISCUSSION TITLE
Only selected drives accessible for input

### DISCUSSION THREAD

**Discussion by ekglover** [vdemichev/DiaNN#821] [Q&A]
When I click Raw to add raw files, only one drive on my computer is visible which is not where I usually store data. Obviously I can just move things there temporarily but I wondered if there was a way round this. Or can DIA-NN only find files stored on the drive that DIA-NN is downloaded to? Thanks for your advice.

**Comment by vdemichev** [comment]
Hi Emily,

Could try supplying the file list via a config file (--cfg command). The GUI dialog is a standard Windows one, cannot change it unfortunately.

Best,
Vadim

**Reply by ekglover** [reply]
I just closed the app and opened it again and I have access to everything now which is great.
